### ISSUE TITLE
Usage of nn.Embedding(max_norm) ?

### ISSUE THREAD

**Issue by sajqavril** [thunlp/OpenKE#236]
Thank you so much for the framework. But I was wandering why you missed the parameter 'max_norm' when init the embedding vector? Is it because that this would bring unexpected lost of leaf node? Or it should save the effort to write the normalization progress.

**Comment by THUCSTHanxu13** [comment]
We  have added the normalization of parameters for each training step, and thus it is not necessary to add extra normalization steps for initialization. Empirically, for some datasets, training models without normalization may achieve better results. Hence, the parameter normalization during model training is optional in the toolkit.

**Comment by sajqavril** [comment]
I see. Thanks a lot.
